benchmarks/distributed/rpc/parameter_server/launcher.py

```diff
@@ -105,15 +106,15 @@ def call_trainers(config, model, train_data, parameter_server_rrefs):
 
 
 def benchmark_warmup(
-    config, model, data, parameter_server_rrefs
+    config, model, data, ps_rrefs
 ):
     if config.ps_count > 0:
         ps_config = config.ps_config
         ps = get_benchmark_ps_map()[str(ps_config["ps_class"])]
-    futs = call_trainers(config, model, data, parameter_server_rrefs)
+    futs = call_trainers(config, model, data, ps_rrefs)
     for fut in futs:
         fut.wait()
```

Comment:
there is an `torch.futures.wait_all()` API for this.

Reply:
okay.